reasoner_pydantic/results.py

```diff
@@ -133,14 +142,21 @@ def __hash__(self) -> int:
     def parse_obj(obj):
         result = parse_obj_as(Result, obj)
         nbindings = HashableMapping.parse_obj(obj["node_bindings"])
-        analyses = HashableSet[Analysis]()
+        analyses = HashableSequence[Analysis]()
         if "analyses" in obj.keys():
             for analysis in obj["analyses"]:
-                analyses.add(Analysis.parse_obj(analysis))
+                analyses.append(Analysis.parse_obj(analysis))
         r = Result(node_bindings=nbindings, analyses=analyses)
         result.update(r)
         return result
 
+    def combine_anlyeses_by_resource_id(self):
+        for analysis in self.analyses:
+            for ana in self.analyses:
```

Comment:
Being very picky, this isn't a great variable name.

Reply:
resolved